backend/shop/views.py

```diff
@@ -82,7 +84,7 @@ def get_profiles(request):
         )
 
     serializer = ShopProfileSerializer(page.object_list, many=True)
-    serialized_data = serializer.data
+    serialized_rows = serializer.data
```

Comment:
one day you'll learn to use https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset and all this code will be simpler :) (but that's a different PR)